zeppelin-web/webpack.config.js

```diff
@@ -267,9 +267,13 @@ module.exports = function makeWebpackConfig () {
     progress: true,
     contentBase: './src',
     setup: function(app) {
-      app.use(
-        '/bower_components/',
-        require('express').static(path.join(__dirname, './bower_components/')));
+      app.use('**/bower_components/',
+        require('express').static(path.resolve(__dirname, './bower_components/')));
+      app.use('**/app/', require('express').static(path.resolve(__dirname, './src/app/')));
```

Comment:
Actually I missed this issue in production mode. 
Yes! I totally agree with @1ambda. I think we need reorganize directory to resolve kind of this issue in correct way.